dvc/types.py

```diff
@@ -10,3 +10,4 @@
 TargetType = Union[List[str], str]
 DictStrAny = Dict[str, Any]
 DictAny = Dict[Any, Any]
+IterableStr = Iterable[str]
```

Comment:
I'm not really sure about whether we want this type alias, IMO `IterableStr` is non-obvious as to what it's supposed to be an alias for since `str` is already an `Iterable`, and it only saves an extra 2 characters vs just using `Iterable[str]`